### PR TITLE
Add a truncate collection dropdown option

### PR DESCRIPTION
**Pull request by samniisan**
# Introduces
- Dropdown option in Collection view, with confirmation modal, to truncate a collection

### CODE REVIEW

File: src/vuex/modules/collection/actions.js [hunk 1]
```diff
@@ -93,5 +93,19 @@ export default {
 
     localStorage.setItem('defaultJsonView', JSON.stringify(indexes))
     return commit(types.SET_COLLECTION_DEFAULT_VIEW_JSON, {jsonView})
+  },
+  [types.CLEAR_COLLECTION] ({state}, {index, collection}) {
```
Comment (jenow) [inline src/vuex/modules/collection/actions.js]: You should unit test this

Comment (samniisan) [inline src/vuex/modules/collection/actions.js]: @jenow Done

**Comment by samniisan** [comment]
@scottinet
- Tested on various browsers, the list will be empties once PR https://github.com/kuzzleio/kuzzle/issues/854 is merged on core side
- There are currently no "success" toast on BO as a design UX choice, checked with @jenow . As long as there is enough visual feedback that the action performed the awaited result (here, the emptied collection), there's no need to add a notification.

**Comment by scottinet** [comment]
Ok :+1:

**Pull request merged**